neutrino.go

```diff
@@ -320,7 +320,8 @@ func (sp *ServerPeer) OnAddr(_ *peer.Peer, msg *wire.MsgAddr) {
 	// helps prevent the network from becoming another public test network
 	// since it will not be able to learn about other peers that have not
 	// specifically been provided.
-	if sp.server.chainParams.Net == chaincfg.SimNetParams.Net {
+	if sp.server.chainParams.Net == chaincfg.SimNetParams.Net ||
```

Comment:
How about we spin these repeated conditionals out into something like `isDevNetwork`? Applies also in the diff for the other file.

Reply:
Made a `isDevNetwork` helper taking a `wire.BitcoinNet`.  Seemed cleanest.